specification/web/resource-manager/Microsoft.Web/stable/2023-01-01/ResourceProvider.json

```diff
@@ -544,6 +544,98 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/virtualNetworkIntegrations": {
+      "post": {
+        "summary": "Retrieve information regarding the provided virtual network integration",
+        "description": "Retrieve information regarding the provided virtual network integration, such as its allocated resources and its allocated IP",
+        "operationId": "VirtualNetworkIntegrations",
+        "parameters": [
+          {
+            "name": "request",
+            "in": "body",
+            "description": "Object contains the subnetResourceId",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VirtualNetworkIntegrationRequest"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/SwiftVirtualNetwork"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetVirtualNetworkIntegrations": {
+            "$ref": "./examples/GetVirtualNetworkIntegrations.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/locations/{location}/purgeUnusedVirtualNetworkIntegration": {
+      "post": {
+        "summary": "Delete orphaned (unused) VNET",
```

Comment:
Should perhaps say VNET integration - the current text sounds like it is the actual VNET that is deleted.